user/tests/wiring/no_fixture/i2c.cpp

```diff
@@ -175,13 +182,14 @@ class SoftWire {
     }
 
     int readAck() {
-        hal_gpio_write(sda_, 1);
+        pinMode(sda_, INPUT_PULLUP);
```

Comment:
Same here. Do we have some problem on P2 specifically with reading the pin state with `OUTPUT_OPEN_DRAIN_PULLUP` ? There shouldn't be any difference between `INPUT_PULLUP` and `OUTPUT_OPEN_DRAIN_PULLUP` unless we are specifically driving the pin to 0.